src/App/Resources/AppResources.resx

```diff
@@ -2643,4 +2640,50 @@ Do you want to switch to this account?</value>
   <data name="InvalidAPIToken" xml:space="preserve">
     <value>Invalid API token</value>
   </data>
+  <data name="BlockAutoFill" xml:space="preserve">
+    <value>Block auto-fill</value>
+  </data>
+  <data name="AutoFillWillNotBeOfferedForTheseURIs" xml:space="preserve">
+    <value>Auto-fill will not be offered for these URIs.</value>
+  </data>
+  <data name="NewBlockedURI" xml:space="preserve">
+    <value>New blocked URI</value>
+  </data>
+  <data name="URISaved" xml:space="preserve">
```

Comment:
@fedemkr Sorry I missed this, looks like `URISaved` made it in twice (see line 2674 below)

Reply:
Nice catch @mpbw2 , will have a PR ready right away